common/containers/Home/index.jsx

```diff
@@ -23,16 +25,41 @@ export class Home extends Component {
     }
   }
 
+  handleNavigateUsers() {
+    if (__CLIENT__) {
+      window.location.href = '/users'
+    }
+  }
+
+  componentDidMount() {
+    this.constructor.fetchData(this.props.dispatch, this.props)
+  }
+
+  static fetchData(dispatch) {
+    dispatch(updateJWT())
+  }
+
   render() {
+    const isAdmin = this.props.isAdmin
     return (
-      <HomeComponent onEditProfile={this.handleEditProfile} onSignOut={this.handleSignOut}/>
+      <HomeComponent onEditProfile={this.handleEditProfile} onSignOut={this.handleSignOut} onNavigateUsers={this.handleNavigateUsers} isAdmin={isAdmin}/>
```

Comment:
Instead of passing down a prop that describes the state of the user `isAdmin`, let's pass down a prop that describes what the state of the component should be. `isAdmin` is used to determine whether or not to show the `Users` button, so we can instead pass a boolean prop like `showUsers`.